.github/workflows/watch.yml

```diff
@@ -0,0 +1,36 @@
+name: Watch
+
+on:
+  schedule:
+    - cron: '0 2,8,14,20 * * *'
```

Comment:
We could miss versions here when multiple versions are tagged within 6 hours, but it's probably not a big problem.